cmd/vault-manager/main.go

```diff
@@ -72,6 +76,17 @@ func main() {
 			log.Fatalln(err)
 		}
 		sleepDuration = sleepDur
+
+		// configure prometheus metrics handler
+		port, _ := os.LookupEnv("METRICS_SERVER_PORT")
```

Comment:
wdyt about having a default?

Reply:
I included one [within the template file](https://github.com/app-sre/vault-manager/pull/81/files#diff-75f81b3202fa823e6bb7873e199f804665cf72cb477af13f8a677d5f8cfced5fL119) if that addresses what you mean